www/base/src/views/WorkerView/WorkerView.tsx

```diff
@@ -52,22 +54,44 @@ export const WorkerView = observer(() => {
         order: "-buildid",
       }
     }));
+  const workersBuildersQuery = useDataApiQuery(
+    () => workersQuery.getRelated(w => w.getBuilders({query: {order: 'name'}}))
+  );
 
   const [workerForActions, setWorkerForActions] = useState<null|Worker>(null);
 
+  const worker = workersQuery.isResolved() && workersQuery.array.length >= 1 ? workersQuery.array[0] : null;
   const topBarActions: TopbarAction[] = [];
-  if (workersQuery.isResolved() && workersQuery.array.length >= 1) {
+  if (worker !== null) {
     topBarActions.push(
       {
         caption: "Actions...",
         variant: "primary",
         action: () => {
-          setWorkerForActions(workersQuery.array[0]);
+          setWorkerForActions(worker);
         },
       });
   }
   useTopbarActions(topBarActions);
 
+  const renderWorkerBuilders = () => {
+    if (worker === null || !workersBuildersQuery.isResolved()) {
+      return <LoadingSpan />;
+    }
+    const builders = workersBuildersQuery.getParentCollectionOrEmpty(worker.id).array;
+    return (
+      <ul className="list-unstyled group-results">
```

Comment:
Maybe reuse BuildersTable a bit to something like:
```
    <Table hover striped size="sm">
      <tbody>
        {builderRowElements}
      </tbody>
    </Table>
```

Reply:
with Table

![image](https://github.com/user-attachments/assets/a1c0eee4-41b3-4572-976c-9ef34e923669)